_posts/2016-08-18-On-Emission.markdown

```diff
@@ -0,0 +1,133 @@
+---
+layout: post
+title: "On our implementation of React Native"
+date: 2016-08-18 12:17
+author: orta
+categories: [tooling, mobile, eigen, node, reactnative]
+series: React Native at Artsy
+---
+
+<center>
+ <img src="/images/emission/emission-logo-artsy.svg" style="height:300px;">
+</center>
+
+I arrived fashionably late to the React Native party in Artsy. I had been a part of our [Auctions Team][auctions_team], where we worked in Swift with [some light-FRP][interstellar]. We were not affected by the 4 months of simultaneous work on moving to React Native, at all.
+
+It was a quiet revolution. We did not have to install `npm`, made zero changes to the code for auctions and the whole app's infrastructure barely changed. What gives? 
+
+Well, first up we weren't planning a re-write, we don't have that kind of luxury and the scope of our app is too big compared to the team working on it. Second, we reused existing dependency infrastructure. Read on to find out what that looks like.
```

Comment:
It took me a while to understand without context what you were going to talk about. So this is about adding react native to a Swift app? Maybe the title of the blog post is inappropriate? Or this paragraph needs to become much clearer.